manage.py

```diff
@@ -7,7 +7,7 @@
 import sys
 
 if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "edxsearch.settings")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings")
```

Comment:
[question]: Why was this done? Why do we have these separate settings modules at all?

Reply:
This was to fix an existing issue with that manage.py file -- it was referencing a non-existent file. AFAIK, you'd only use this to run a dev server locally, not for anything real.